wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests/HttpGetWithRedirectTest/content.txt

```diff
@@ -0,0 +1,9 @@
+!2 Http GET without follow redirect
+
+Performs HTTP GET adding values as parameters in the query string.
+
+|script         |xml http test                                                       |
+|get from       |${URL}         | with redirect       | false        |
```

Comment:
Do you know a public site that actually sends a redirect we could use here.
I prefer all the examples to be green...